pkglistgen/tool.py

```diff
@@ -578,7 +578,8 @@ def update_and_solve_target(self, api, target_project, target_config, main_repo,
                                 project, scope, force, no_checkout,
                                 only_release_packages, stop_after_solve):
         self.all_architectures = target_config.get('pkglistgen-archs').split(' ')
-        self.repos = self.expand_repos(project, main_repo)
+        ignore_repos = set(target_config.get('pkglistgen-ignore_repos').split(' '))
+        self.repos = [ r for r in self.expand_repos(project, main_repo) if r[0] != project or r[1] not in ignore_repos ]
```

Comment:
Is it intentional that the `ignore_repos` list doesn't influence recursive expansion?

If you have e.g. `projectA/standard -> projectB/bootstrap_copy -> projectC/standard`, then it would solve against `projectA/standard -> projectC/standard`

Reply:
project C doesn't contain any packages in the factory case so doesn't matter.